src/Microsoft.AspNet.Mvc.Core/ModelBinding/MutableObjectModelBinder.cs

```diff
@@ -359,7 +365,7 @@ protected virtual object GetModel([NotNull] ModelBindingContext bindingContext)
                 var modelMetadataPredicate = context.ModelMetadata.PropertyBindingPredicateProvider?.PropertyFilter;
 
                 return
-                    context.PropertyFilter(context, propertyName) &&
+                    (context.PropertyFilter == null || context.PropertyFilter(context, propertyName)) &&
                     (modelMetadataPredicate == null || modelMetadataPredicate(context, propertyName));
```

Comment:
This can be null now. It's easier to just allow it to be null than to try and force it be initialize/copied everywhere.